test/ndb_api_test.go

```diff
@@ -22,27 +22,27 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/nutanix-cloud-native/ndb-operator/api/v1alpha1"
-	"github.com/nutanix-cloud-native/ndb-operator/ndbclient"
+	"github.com/nutanix-cloud-native/ndb-operator/ndb_api"
+	"github.com/nutanix-cloud-native/ndb-operator/ndb_client"
 )
 
 // func TestGetAllDatabases(t *testing.T) {
 
 // 	server := util.GetServerTestHelper(t, "/databases")
 // 	defer server.Close()
-// 	ndbclient := ndbclient.NewNDBClient("username", "passwdord", server.URL)
-// 	value := GetAllDatabases(ndbclient)
```

Comment:
Do you need this comment or ok to remove?

Reply:
That was a miss, thanks for catching this. Removed.